doc/packages/krita.section.md

```diff
@@ -0,0 +1,27 @@
+# Krita {#sec-krita}
+
```

Comment:
Maybe worth to also mention python plugin in this doc to avoid confusion. Ask user to follow https://docs.krita.org/en/user_manual/python_scripting/install_custom_python_plugin.html or simply unzip it to `$HOME/.local/share/krita/pykrita/`